app/models/embed/purl_json_loader.rb

```diff
@@ -56,6 +57,10 @@ def stanford_only_unrestricted
       json.dig('access', 'download') == 'stanford'
     end
 
+    def stanford_only_no_download
+      json.dig('access', 'download') == 'none' && json.dig('access', 'view') == 'stanford'
+    end
+
```

Comment:
Cocina JSON: A new restriction type we are catching (currently only for the PDF viewer, but could be elsewhere).